pkg/cluster/manifests/values.yaml

```diff
@@ -12,9 +12,14 @@ global:
   masterNodeCount: 1
   nodeType: t3.large
   kubeconfigSecretKeyName: value
+  clusterAutoscalerEnabled: false
+  clusterAutoscalerMinNodes: "1"
+  clusterAutoscalerMaxNodes: "9"
```

Comment:
Shouldn't these be integers?

Reply:
These are used to create 2 annotations on the MachineDeployment, see https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/clusterapi#enabling-autoscaling
Since annotation values are strings, I kept them consistent here. Switching them to integers might also work, but haven't tested.